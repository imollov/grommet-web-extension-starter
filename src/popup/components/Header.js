import React from 'react';
import { Header, Heading } from 'grommet';
import { Gremlin } from 'grommet-icons';

export default () => (
  <Header background="brand" justify="center">
    <Gremlin />
    <Heading size="small">Sample Extension</Heading>
  </Header>
);
