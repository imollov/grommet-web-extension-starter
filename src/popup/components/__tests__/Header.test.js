import React from 'react';
import { shallow } from 'enzyme';

import Header from '../Header';

describe('Header Component', () => {
  it('smoke test', () => {
    const wrapper = shallow(<Header />);

    expect(wrapper).toExist();
  });

  it('should match snapshot', () => {
    const wrapper = shallow(<Header />);

    expect(wrapper).toMatchSnapshot();
  });
});
