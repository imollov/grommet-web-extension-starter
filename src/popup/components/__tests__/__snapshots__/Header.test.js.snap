// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component should match snapshot 1`] = `
<Header
  background="brand"
  justify="center"
>
  <Gremlin />
  <Heading
    level={1}
    responsive={true}
    size="small"
  >
    Sample Extension
  </Heading>
</Header>
`;
